hook/rpm/SPECS/nvidia-container-runtime-hook.spec

```diff
@@ -23,7 +23,10 @@ Requires: libnvidia-container-tools >= 0.1.0, libnvidia-container-tools < 2.0.0
 Provides a OCI hook to enable GPU support in containers.
 
 %prep
-cp %{SOURCE0} %{SOURCE1} %{SOURCE2} %{SOURCE3} %{SOURCE4} .
+cp %{SOURCE0} %{SOURCE1} %{SOURCE4} .
+%if 0%{?fedora} || 0%{?rhel_version} || 0%{?centos_version}
```

Comment:
Can you detail why this is needed ?

Reply:
well, as you can see further below, openSUSE doesn't need the files, so I also didn't want to copy them in.

Reply:
in the latest version I'm copying them again to keep it simpler